sdk/maps/azure-maps-render/copyrightcaptionoutput.txt

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What are all these files? I assume they are log files and as such should be deleted.